bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/custom/CommonWidgetsDPIChangeHandlers.java

```diff
@@ -11,9 +11,10 @@
  * Contributors:
  *     Yatta Solutions - initial API and implementation
  *******************************************************************************/
-package org.eclipse.swt.internal;
+package org.eclipse.swt.custom;
```

Comment:
Since we were using some of the package protected fields (also will use them in upcoming PRs) and also we wanted to move this new class to win32 packages anyway.

Reply:
This class doesn't add any value to "normal" users.
Move it back to internal and change the handleCComboDPIChange instead so that it delegates everything to CCombo (1 call) instead of doing 4 calls and passing the fields of CCombo (that's a code smell: https://refactoring.guru/smells/inappropriate-intimacy)

Reply:
What should it delegate to CCombo? CCombo is a common widget for all OS, we e.g. cannot use the windows specifics like the DPIZoomChangeRegistry in CCombo

Reply:
As discussed in the meeting, we have delegated everything to CCombo by creating adapt method and passing the lamba function to apply changes internally in CCombo class, @fedejeanne @akoch-yatta